lib/core/plugin.rb

```diff
@@ -61,9 +63,8 @@ def write_script(filename)
   end
 
   def load_passive
```

Comment:
Can do, I thought I originally needed it for loading labels. But now I'm not sure if we'll need it? Maybe we need to sit down and do a project review after the pending merges to decide on what code blocks we need to keep or can based on what we know now.